_specs/ecip-_.md

```diff
@@ -0,0 +1,103 @@
+---
+ecip: TBD
+title: The Meta Fork ECIP: Establishing _Fork_ type ECIPs and their specifications and processes
+author: Mr. Meows D. Bits
+discussions-to: TBD
+status: WIP
+type: Meta
+created: 2019-11-29T14:17:42-05:00
+resolution: TBD
```

Comment:
What does this field mean?

Reply:
No idea. Resolved with https://github.com/ethereumclassic/ECIPs/pull/224/commits/aaf75959895341c82e900081f35611886d055fe9